src/netutil.rs

```diff
@@ -1,17 +1,22 @@
 //! The `netutil` module assists with networking
 
+#[cfg(target_os = "linux")]
 use libc::{
     c_void, iovec, mmsghdr, recvmmsg, sockaddr_in, socklen_t, time_t, timespec, MSG_WAITFORONE,
 };
+use nix::sys::socket::setsockopt;
 use nix::sys::socket::sockopt::{ReuseAddr, ReusePort};
-use nix::sys::socket::{setsockopt, InetAddr};
+#[cfg(target_os = "linux")]
+use nix::sys::socket::InetAddr;
 use packet::Packet;
 use pnet_datalink as datalink;
 use rand::{thread_rng, Rng};
 use reqwest;
 use socket2::{Domain, SockAddr, Socket, Type};
+#[cfg(target_os = "linux")]
 use std::cmp;
 use std::io;
+#[cfg(target_os = "linux")]
```

Comment:
I think you can move these uses into the function, might look cleaner

Reply:
That's true. I didn't realize that "use" can be moved within the function. I'll push a patch.